main.py

```diff
@@ -0,0 +1,13 @@
+from fastapi import FastAPI
+from city.router import router as city_router
+from temperature.router import router as temperature_router
+
```

Comment:
[change request] Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports)